packages/tracing/src/browser/web-vitals/lib/initMetric.ts

```diff
@@ -17,13 +17,12 @@
 import { Metric } from '../types';
 import { generateUniqueID } from './generateUniqueID';
 
-export const initMetric = (name: Metric['name'], value = -1): Metric => {
+export const initMetric = (name: Metric['name'], value?: number): Metric => {
   return {
     name,
-    value,
+    value: typeof value === 'undefined' ? -1 : value,
```

Comment:
```suggestion
    value: value ?? -1,
```

We use this syntax in tracing package already so it should be ok